seed/python-sdk/alias/src/seed/version.py

```diff
@@ -0,0 +1,4 @@
+
+from importlib import metadata
+
+__version__ = metadata.version("fern_alias")
```

Comment:
will this normally keep the version string?

Reply:
Yeah, this is meant to evaluate to the version within the toml file